client/assets/js/main.js

```diff
@@ -2,36 +2,43 @@
 
 import Board from './board';
 import server from './sendToServer.js';
+import createCanvasGame from './createCanvasGame.js';
 
 document.addEventListener('DOMContentLoaded', function () {
 
-	let board = new Board();
+	let context = createCanvasGame()
+	let board = new Board(context);
 
 
     // TEMP: CREATE TWO SNAKES FOR TEST PURPOSES
-	board.newSnake(0, 60, 'Jean-Mich');
-	board.newSnake(420, 330, 'Xx-JeanKevin33-xX');
+	// board.newSnake(0, 60, 'Jean-Mich');
+	// board.newSnake(420, 330, 'Xx-JeanKevin33-xX');
     // END TEMP
 
-    server.on('connection', function(){
+	server.on('connection', function(){
     	/* Random place on board for testing purpose */
     	let long = Math.floor(Math.random() * 1000);
     	let lat = Math.floor(Math.random() * 500);
     	board.newSnake(long, lat);
-    });
+	});
 
-    server.on('disconnection', function(){
+	server.on('disconnection', function(){
     	// Destroy Snake
-    });
+	});
+
+server.on('new apple', function(data){
```

Comment:
weird indent. Use eslint. Be careful maybe eslint is currently configured to only check client side files